components/bokeh/__init__.py

```diff
@@ -0,0 +1,67 @@
+from bokeh.plotting import figure
+from bokeh.models import ColumnDataSource
+from bokeh.plotting import figure, show
+from bokeh.transform import factor_cmap
+from bokeh.models import Select
+from bokeh.models.callbacks import CustomJS
+from bokeh.layouts import column
+import numpy as np
+import seaborn as sns
+
+def generate_colors(input_df, feature):
+    pal = sns.color_palette()
+    color = factor_cmap(feature, palette=pal.as_hex(), factors=np.array(input_df[feature].unique()))
+
+    return color
+
+def interactive_circle_plot(input_df, x_lab, y_lab, feature):
+    input_df['legend'] = input_df[feature]
+    source = ColumnDataSource(input_df)
+
+    TOOLTIPS = [
+        ("index", "$index"),
+        ("(x,y)", "($x, $y)"),
+        ("feature", '@'+feature),
+    ]
+
+    n = len(input_df[feature].unique())
+    p = figure(width=800, height=400, tooltips=TOOLTIPS,x_axis_label=x_lab, y_axis_label=y_lab)
```

Comment:
It would be good if the figure would autosize itself to fit within the flex container instead of having an explicit size